doc/production/README.md

```diff
@@ -0,0 +1,43 @@
+# Production
+
+Nowadays, you can run JavaScript code in several different environments, some of them with their own particularities. Moreover, you can use `js-libp2p` for a wide range of use cases. Different environments and different use cases mean different configurations and challenges in the network.
+
+Libp2p nodes can vary from nodes behind an application, to infrastructure nodes that enable the network to operate and to be efficient. In this context, the Libp2p project provides public infrastructure to boost the network, enable nodes connectivity and improve constrained nodes performance. This public infrastructure should be leveraged for learning the concepts and experimenting. When an application on top of libp2p aims to move into production, its own infrastructure should be setup as the public nodes will be intensively used by others and its availability is not guaranteed.
+
+This guide aims to guide you from using the public infrastructure into setting up your own.
+
+## Table of Contents
```

Comment:
I would reframe this guide. Right now it is focused on the services and less on what the user actually needs. I would think about it in terms of flows. Something like:

- Joining the Network
  - How do I join the public network, how can I run my own bootstrap nodes? JS isn't currently great for being bootstrap nodes, because we need to improve the DHT first to be an effective server.
- Connecting to browser nodes
  - This is where webrtc-star and relays come in
- Querying the network from the browser
  - Delegates go here. In theory the DHT _can_ work, it's just slow and requires WSS or webrtc-star support on all dialable nodes, we just really don't recommend it and if you're using the public dht it's really not going to work.

Reply:
Yes, I think this makes sense

Reply:
I added a "Others" section where I included the SSL subsection.  SSL and other things will be transversal to the flows described